Mixmatch/streetview_dataset/parse_data_to_tfrecord_lib_test.ipynb

```diff
@@ -0,0 +1,239 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Parse_data_to_tfrecord_library Test\n",
+    "\n",
+    "This file consists several function test for the functions in the Parse_data_to_tfrecord_lib file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from parse_data_to_tfrecord_lib import img_to_example, read_tfrecord, generate_tfexamples_from_detections, batch_read_write_tfrecords\n",
+    "from PIL import Image  # used to read images from directory\n",
+    "import tensorflow as tf\n",
+    "import os\n",
+    "import io\n",
+    "import IPython.display as display"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test function img_to_example():\n",
+    "The function reads in image files and outputs tf.examples. The output should store the information feeded into the function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "IMG_PATH = './TC11/svt1/img/19_00.jpg'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    img = Image.open(IMG_PATH, \"r\")\n",
+    "except Exception as e:\n",
+    "    print(e)\n",
+    "    print(IMG_PATH + \" is not valid\")\n",
+    "\n",
+    "example = img_to_example(img, label=0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Testing\n",
+    "# The label feature should be a int64_list with value of {value: 0}\n",
+    "print(example.features.feature['label'])\n",
+    "\n",
+    "# The image feature should be a bytes_list storing the image\n",
+    "print(example.features.feature['image'].SerializeToString())"
```

Comment:
Is it possible to compare this field with the original `img`?

Reply:
Done. Compared by decode image to numpy array and compare the flattened value and the original value.